stable/prometheus/values.yaml

```diff
@@ -400,6 +400,9 @@ server:
 ## alertmanager ConfigMap entries
 ##
 alertmanagerFiles:
+  # ConfigMap override where fullname is {{.Release.Name}}-{{.Values.alertmanagerFiles.configMapOverrideName}}
+  configMapOverrideName: ""
```

Comment:
Let's move this up to the `alertmanager` section. Also, wrap the contents of `templates/alertmanager-configmap.yaml` in a conditional so that if `alertmanager.configMapOverrideName` is set, we don't create an unnecessary resource. Then add a note here (and in the README) to that effect.